ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -575,6 +591,29 @@ import { DraggableListBehavior } from '../nuxeo-draggable-list-behavior.js';
       });
     }
 
+    _itemsListener() {
+      if (this.items.length >= 0) {
+        afterNextRender(this, () => this._alignHeaderRow());
```

Comment:
Without taking a spin, I would say that we may be running this alignment more times than expected.
There seem to be a lot of methods that would trigger this observer:
- _moveItemUpward
- _moveItemDownward
- _deleteEntry
- _createEntry
- _validateEntry

Also, all those are executing `this.notifyResize()`, which seems to trigger `this._resizeCellContainers` (https://github.com/nuxeo/nuxeo-elements/blob/db4372d6ef4af3cc7e90bb5072429d9387fa81e5/ui/nuxeo-data-table/iron-data-table.js#L570)

Despite not being critical, we may improve the flow, and consequently the performance, here.

Reply:
Indeed. I think with the new solution we can optimise this aspect.